packages/server/cloud/nlp/package.json

```diff
@@ -24,16 +26,16 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@eslint/js": "8.57.0",
-    "@types/express": "^4.17.13",
+    "@eslint/js": "9.21.0",
+    "@types/express": "4.x",
     "@types/node": "20.14.8",
     "@types/uuid": "^8.3.1",
-    "concurrently": "8.2.2",
-    "eslint": "8.57.0",
-    "eslint-plugin-node": "^11.1.0",
+    "concurrently": "9.0.1",
+    "eslint": "9.21.0",
+    "eslint-plugin-n": "17.16.2",
```

Comment:
was not used .. older plugin is unmaintained, the replacement is an eslint community fork